packages/client/hmi-client/src/model-representation/petrinet/nested-petrinet-renderer.ts

```diff
@@ -65,28 +69,30 @@ export class NestedPetrinetRenderer extends PetrinetRenderer {
 			.attr('stroke-width', 1)
 			.style('cursor', 'pointer');
 
-		species.each((d, idx, g) => {
-			const nestedNodes = this.nestedMap[d.id] ?? [];
-			const nestedNodesLen = nestedNodes.length;
-			for (let i = 0; i < nestedNodesLen; i++) {
-				// skip nodes that can't be laid out for now
-				if (nestedNodesLen >= CIRCLE_PACKING_CHILD_NORMALIZED_RADII.length) {
-					continue;
-				}
-				const nodeId = nestedNodes[i];
-				const parentRadius = 0.55 * d.width;
+		const renderNestedNodes = (node, parentRadius, parentX, parentY, g, idx) => {
```

Comment:
Would appreciate a bit of comments here to describe what this function is doing.

Would appreciate if you can give concrete types of the function parameters.

Reply:
let me know if the typings and comment look good